unlock-app/src/components/content/LogContent.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import Head from 'next/head'
+import Layout from '../interface/Layout'
+import CreatorAccount from '../creator/CreatorAccount'
+import CreatorLog from '../creator/CreatorLog'
+import BrowserOnly from '../helpers/BrowserOnly'
+import GlobalErrorConsumer from '../interface/GlobalErrorConsumer'
+import { pageTitle } from '../../constants'
+import withConfig from '../../utils/withConfig'
+import * as UnlockTypes from '../../unlock'
+
+interface Props {
+  account: UnlockTypes.Account
+  network: UnlockTypes.Network
+  transactionFeed: UnlockTypes.Transaction[]
+  transactionMetadataMap: UnlockTypes.TransactionMetadataMap
+}
+
+export const LogContent = (props: Props) => {
```

Comment:
can we destructure in the function declaration? (don't need to change this, just wondering about the complexity level of that)

Reply:
it becomes a mess with TS ;)

Reply:
Haha yup, you can destructure, but the type declaration gets very ugly.

Reply:
Oooh, did you mean can we do
```
({ account, network, transactionFeed, transactionMetadataMap}: Props) => ...
```

🤔 I didn't try, but I'm going to now.